### PR TITLE
feat: add member and address components 

### PR DESCRIPTION
**Pull request by domw30**
- add member component that takes children and determines whether the child is a link or text.
- add address component that returns a string or async text
- add tests, stories, styles for both

### CODE REVIEW

**Comment by domw30** [comment]
Not sure if it would make sense to install `zApp-utils` to `zUI` for use of `truncateAddress` within this PR, or just pass a truncated address to these components from the `zApp`.

`zUI` is a peer dependency of `zApp-utils`. We'd have to update `zUI` package version in `zApp-utils`  to the latest (as it is currently behind) and some other packages for the install to run. This suggests to me it would be painful to keep track of package versions up to date to avoid any breaks.

**Comment by colbr** [comment]
> Not sure if it would make sense to install `zApp-utils` to `zUI` for use of `truncateAddress` within this PR, or just pass a truncated address to these components from the `zApp`.
> 
> `zUI` is a peer dependency of `zApp-utils`. We'd have to update `zUI` package version in `zApp-utils` to the latest (as it is currently behind) and some other packages for the install to run. This suggests to me it would be painful to keep track of package versions up to date to avoid any breaks.

Hmm yes it would definitely be better if we could pass in an address and it truncates internally. Maybe `truncateAddress` is a function which belongs in zUI, rather than zApp utils?

**Comment by domw30** [comment]
> 

Yeah nice idea. I've also pulled in `truncateDomain` as at some point we'll be adding `Wizard.NFTDetails` which will require this formatting, as with other components

**Comment by domw30** [comment]
Also, I'm wn

**Review by colbr — DISMISSED** [review]

File: src/components/Address/Address.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+export const Address = ({ className, address }: AddressProps) => {
+  if (typeof address === 'string') {
+    return <span className={className}>{truncateAddress(address)}</span>;
+  }
+
+  const truncatedAddress = address?.text ? truncateAddress(address.text as string) : address.text;
+
+  return (
+    <SkeletonText
```
[the remark this answers is not in the document]
Comment (domw30) [reply src/components/Address/Address.tsx:27]: Good shout, updated!

**Review by colbr — APPROVED** [review]

**Pull request merged**